pkgs/top-level/python-packages.nix

```
@@ -12271,6 +12271,7 @@ self: super: with self; {
   pytest-rerunfailures = callPackage ../development/python-modules/pytest-rerunfailures { };
 
   pytest-resource-path = callPackage ../development/python-modules/pytest-resource-path { };
+  pytest-responses = callPackage ../development/python-modules/pytest-responses { };
```

nit: leave a space between packages in this file to help prevent future merge conflicts
```suggestion

  pytest-responses = callPackage ../development/python-modules/pytest-responses { };
```

---

Done. Thanks!